views/nexthome.tmpl

```diff
@@ -2,253 +2,350 @@
+                            title='{"object": "Ticket", "total": "{{.Total}}", "vout": "{{.VoutCount}}", "vin": "{{.VinCount}}"}'>
+                            <a class="block-element-link" href="/tx/{{.TxID}}"></a>
+                        </span>
+                        {{end}}
+                        {{if gt $ticketsCount 50}}
+                        <!-- show total number of tickets -->
+                        <span class="block-ticket" style="flex-grow: 10; flex-basis: 50px;" title="Total of $ticketsCount tickets">
+                            <a class="block-element-link" href="#">+ {{intSubtract $ticketsCount 30}}</a>
+                        </span>
+                        {{end}}
+                        {{range .Revocations}}
+                        <span class="block-rev"
+                            title='{"object": "Revocation", "total": "{{.Total}}", "vout": "{{.VoutCount}}", "vin": "{{.VinCount}}"}'>
+                            <a class="block-element-link" href="/tx/{{.TxID}}"></a>
+                        </span>
+                        {{end}}
+                        {{$ticketsRevsCount := intAdd $ticketsCount (len .Revocations)}}
+                        {{range (zeroSlice (intSubtract 20 $ticketsRevsCount))}}
+                        <span title="Empty ticket slot"></span>
+                        {{end}}
+                        </div>
+
+                        <div class="block-transactions" style="flex-grow: 1">
+                        {{$txs := .Transactions}}
+                        {{$txCount := len .Transactions}}
+                        {{if gt $txCount 50}} {{$txs = clipSlice .Transactions 30}} {{end}}
+                        {{range $txs}}
+                        <span class="block-tx" style="flex-grow: {{.Total}}"
+                            title='{"object": "Transaction", "total": "{{.Total}}", "vout": "{{.VoutCount}}", "vin": "{{.VinCount}}"}'>
+                            <a class="block-element-link" href="/tx/{{.TxID}}"></a>
+                        </span>
+                        {{end}}
+                        {{if gt $txCount 50}}
+                        <!-- show total number of tickets -->
+                        <span class="block-ticket" style="flex-grow: 10; flex-basis: 50px;" title="Total of $txCount transactions">
+                            <a class="block-element-link" href="#">+ {{intSubtract $txCount 30}}</a>
+                        </span>
+                        {{end}}
                         </div>
                     </div>
-                    {{end}}
-  		        </div>
-                <a class="no-underline" href="/blocks"><div class="showmore"><p>View BlockChain History</p></div></a>
+                {{end}}
+                </div>
+                {{end}}
             </div>
+            <a class="no-underline" style="display: block; margin-top: 25px;" href="/blocks"><div class="showmore"><p>View BlockChain History</p></div></a>
```

Comment:
Try to use Bootstrap classes instead of inline styles, whenever possible.

It improves render performance (inline styles cause repaints as the browser gets to them)
It also improves versatility and consistency of codebase when combined together with other utility classes to produce outcomes. 

Also, it's also a very effective way to learn the framework. Whenever you notice yourself wanting to write a inline style, or declare a new classes for minor or one-off position/layout reasons, go check the bootstrap docs and see if you can't to it declaratively by composing a set of existing utility classes to achieve the end goal.  You will be amazed to find you often don't need to write any css at all.

In this case `d-block` will apply `"display: block;`

IMO that inline padding can most likely be handled via Boostrap classes as well, so that we can easily adjust it for responsive layouts without having to write any css.